CHANGELOG.md

```diff
@@ -34,7 +37,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - LORIX One gateway documentation.
 - Display own user name instead of ID in Console if possible.
 - Option to hide rarely used fields in the Join Settings step (end device wizard) in the Console.
-- Payload formatter documentation.
```

Comment:
This did land in 3.8.5 right? I see documentation added here, but we shouldn't be changing the release notes after a release, unless it's obviously wrong.

Reply:
No, it was merged after 3.8.5, and probably must have slipped in the CHANGELOG https://github.com/TheThingsNetwork/lorawan-stack/pull/2836